proto/auth.proto

```diff
@@ -6,7 +6,7 @@ option go_package = "github.com/momentohq/client-sdk-go;client_sdk_go";
 option java_multiple_files = true;
 option java_package = "momento.auth";
 
-package auth;
+//package auth;
```

Comment:
Why did we comment out all the package names? JC want to double check meant to leave that in?